src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
+  let tempValueRobot = {};
+
+  tempKyesRobot = Object.keys(robot);
```

Comment:
There is no need to initialize `tempKyesRobot` with an empty object on line 11 since you are assigning it a new value here. You can declare and assign it in one line.